docker/docker-compose-test.yaml

```diff
@@ -1,23 +1,23 @@
 # export COMPOSE_DOCKER_CLI_BUILD=0 # some machines need for smooth --build
-# docker-compose -f docker-compose-test.yaml up --build
+# docker-compose -f ./docker/docker-compose-test.yaml up --build
 # docker exec -it workflow-session_workflow_1 /bin/bash
-# docker-compose -f docker-compose-test.yaml down
+# docker-compose -f ./docker/docker-compose-test.yaml down
 
 version: "2.4"
 x-net: &net
   networks:
       - main
 services:
-  db:
+  session-test-db:
     <<: *net
     image: datajoint/mysql:5.7
     environment:
       - MYSQL_ROOT_PASSWORD=simple
-  workflow:
+  session-test-workflow:
     <<: *net
     build:
-      context: ../
-      dockerfile: ./workflow-session/Dockerfile.test
+      context: ../../
+      dockerfile: ./workflow-session/docker/Dockerfile.test
     env_file: .env
     image: workflow_session_test:0.0.0b2
     environment:
```

Comment:
Would it work if you stored this variable within the `.env` file?  In my case, I was using the `dj_local_conf.json` to define my host.

As a related aside, in the future I would like to merge the `test` and `dev` Docker and Compose files.

Reply:
Sure - this section could be dropped altogether if we were certain this info was in the environment file ported into the container. In my case, having the conflicting definitions of `DH_HOST` caused some issues where the pytest would automatically fail because it would try connecting to `docker_db_1`

Merging `test` and `dev` would make sense to me if there was an easy way to compose up with a flag that dictated whether or not to run the pytests. I've been doing dev by composing the `test` once, and then using the docker gui to only run the db, connecting to it from the terminal of my host machine. With the dockerized ipython environment, I don't have the same history space to draw from when composing commands